lib/cadet/assessments/assessments.ex

```diff
@@ -9,8 +9,11 @@ defmodule Cadet.Assessments do
 
   alias Timex.Duration
 
-  alias Cadet.Assessments.Assessment
-  alias Cadet.Assessments.Question
+  alias Cadet.Accounts.User
+  alias Cadet.Assessments.{Answer, Assessment, Question, Submission}
+
+  # TODO: Change once these controllers are shipped
```

Comment:
I think you can remove this TODO now?

Reply:
Yeah woops I forgot to remove when I made the changes.